From cb62706222035a79ceef9f59b83868d4931ea1d7 Mon Sep 17 00:00:00 2001
From: Liang Geng <geng.161@osu.edu>
Date: Wed, 12 Jan 2022 20:21:48 -0500
Subject: [PATCH] adapt for tensorflow

---
 BUILD | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/BUILD b/BUILD
index 1124321..7193b04 100644
--- a/BUILD
+++ b/BUILD
@@ -50,7 +50,7 @@ GTEST_MAIN = select({
 # ZLIB configuration
 ################################################################################
 
-ZLIB_DEPS = ["@zlib//:zlib"]
+ZLIB_DEPS = ["@zlib"]
 
 ################################################################################
 # Protobuf Runtime Library
@@ -199,6 +199,7 @@ cc_library(
     copts = COPTS,
     includes = ["src/"],
     linkopts = LINK_OPTS,
+    alwayslink = 1,
     visibility = ["//visibility:public"],
 )
 
@@ -271,6 +272,7 @@ cc_library(
     copts = COPTS,
     includes = ["src/"],
     linkopts = LINK_OPTS,
+    alwayslink = 1,
     visibility = ["//visibility:public"],
     deps = [":protobuf_lite"] + PROTOBUF_DEPS,
 )
-- 
1.8.3.1

